db/migrate/20170217194936_add_properties_to_physical_servers.rb

```diff
@@ -0,0 +1,16 @@
+class AddPropertiesToPhysicalServers < ActiveRecord::Migration[5.0]
+  def change
+    add_column :physical_servers, :hostname, :string
+    add_column :physical_servers, :ipv_4_addresses, :string, :array => true, :default => []
+    add_column :physical_servers, :ipv_6_addresses, :string, :array => true, :default => []
+    add_column :physical_servers, :mac_addresses, :string, :array => true, :default => []
```

Comment:
I think we have everything software defined hanging off of either operating_system or guest_devices (which hangs off of hardware)